docs/source/federation-spec.md

```diff
@@ -413,3 +413,41 @@ type User @key(fields: "id") {
   name: String @override(from: "SubgraphA")
 }
 ```
+
+### `@tag`
+
+```graphql
+directive @tag(from: String!) on FIELD_DEFINITION | INTERFACE | OBJECT | UNION | ARGUMENT_DEFINITION | SCALAR | ENUM | ENUM_VALUE | INPUT_OBJECT | INPUT_FIELD_DEFINITION
+```
+
+The `@tag` directive is used to provide a mechanism for applying arbitrary string metadata to the fields and types of a schema. This metadata is potentially useful throughout the schema’s lifecycle, including, but not limited to, processing, static analysis, and documentation.
```

Comment:
should we also link to https://www.apollographql.com/docs/studio/contracts as that is the primary use case?